packages/blob/package.json

```
@@ -19,7 +19,10 @@
   "exports": {
     ".": {
       "types": "./dist/src/lib.d.ts",
-      "browser": "./src/lib.js",
+      "browser": {
+        "require": "./dist/lib.cjs",
+        "import": "./src/lib.js"
```

```suggestion
        "default": "./src/lib.js"
```

then it'll be the same as before if `require` condition isn't provided. either approach _should_ work fine, though 👍